src/components/ui/form-field.tsx

```diff
@@ -0,0 +1,24 @@
+import { InputErrors } from "./errors";
+import { Label } from "./label";
+
+export default function FormField(props: {
+  label?: string;
+  required?: boolean;
+  errors?: string[];
+  children: React.ReactNode;
+}) {
+  const { label, errors, children, required } = props;
+
+  return (
+    <>
+      {label && (
+        <Label className="mb-2">
+          {label}
+          {required && <span className="text-red-500">*</span>}
+        </Label>
+      )}
+      {children}
+      <InputErrors errors={errors} />
+    </>
+  );
+}
```

Comment:
This would be conflicting with shadcn's form-field. Why not just shadcn's form elements directly?

<img width="1131" alt="image" src="https://github.com/user-attachments/assets/551a77a6-e038-461d-aa8f-c18729899d25" />

Reply:
I saw the current form implementation, but it looked like a lot of refactoring and code redundancy. Wanted to get these changes done quick, can switch to form component later. For now the temporary form field component is just clubbing label and error. It can be updated to have shadcn form components.

Reply:
Maybe call it InputWithError?

Reply:
@bodhish changed name